### ISSUE TITLE
PUPIL not being correctly populated in wfss simulator

### ISSUE THREAD

**Issue by bhilbert4** [spacetelescope/mirage#319]
Pupil value in my NIRCam WFSS simulation (uncal file) is being set to CLEAR when it should be GRISMR/GRISMC.

**Comment by bhilbert4** [comment]
Ah, but this was in yaml files that I had edited manually, so I should have changed the pupil value myself in the Mirage inputs.

The underlying problem is that when I ran CAR-35 (prop 1409) through the yaml generator, it failed to recognize the WFSS exposures as WFSS and instead set them to imaging. I need to take a more in-depth look at this, as I have run WFSS proposals through the yaml_generator successfully in the past.
